pkg/builders/builders.go

```diff
@@ -19,3 +21,18 @@ const OpsrcOwnerNameLabel string = "opsrc-owner-name"
 // that are owned by the OperatorSource. When this label is set, the reconciler
 // should handle these resources when the OperatorSource is deleted.
 const OpsrcOwnerNamespaceLabel string = "opsrc-owner-namespace"
+
+// GetLabelMap returns a map with either the csc or opsrc owner name and
+// namespace labels depending what kind is the owner
+func GetLabelMap(name, namespace, owner string) map[string]string {
```

Comment:
I'm still wondering about why is it "opsrc or not". Why not also check for `v2.CatalogSourceConfigKind` and if it's neither just return an empty map?

Reply:
You could use a switch statement for this.